nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,70 @@
+const { response } = require("express");
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+// www.javascripttutorial.net/object/convert-an-object-to-an-array-in-javascript/
+// https://stackoverflow.com/questions/32817027/check-if-an-array-contains-only-numeric-values
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce
+
+const extractQueryNumbers = (object) => {
+  const queryNumbers = Object.values(object);
+  const unnestQueryNumbers = queryNumbers.flat();
```

Comment:
```suggestion
  const unnestQueryNumbers = Object.values(object).flat();
```
a little bit shorter